db/app/app-tests/app.test.js

```diff
@@ -89,3 +89,27 @@ describe("/api/articles/:article_id", () => {
       });
   });
 });
+
+describe('/api/getArticles', () => {
+  test('200: Should return an array of all article objects with multiple properties, sorted by date and no body property', () => {
+    return request(app)
+    .get("/api/articles")
+    .expect(200)
+    .then(({body})=> {
+      expect(body).toHaveLength(13)
+      body.forEach((article) => {
+        console.log(article)
+        expect(article).toMatchObject({
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: expect.any(Number),
+          topic: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+          comment_count: expect.any(String)
+        })
+      })
+    })
+  });
```

Comment:
You should check also that the articles don't have the key of body, it's in the (very clear and good) description but not tested for here
They could currently still have the key of body and still pass this test 🕵️ 
The description also mentions that they're sorted by `created_at` but this isn't tested for either
I recommend `jest-sorted` for this